Makefile

```diff
@@ -1,4 +1,4 @@
-KUTTL_VERSION=0.1.0
+KUTTL_VERSION=0.5.0
```

Comment:
Let's use the latest version of kuttl, 0.6.1
```suggestion
KUTTL_VERSION=0.6.1
```

Reply:
should be updated in `dev` branch and not in this PR